### DISCUSSION TITLE
Global variables

### DISCUSSION THREAD

**Discussion by edzikk** [grafana/grafana#57623] [General]
There has been a long-standing need to implement global variables or similar functionality. Often it is necessary to make a change that will affect all panels and dashboards in which this variable or setting is applied. Similar to local variables, only acting globally.

I am sure that this functionality will be useful to many developers and companies, as it will save a lot of time when setting up a large number of panels.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
